easybuild/easyconfigs/c/Caffe/Caffe-rc3-foss-2016a-CUDA-7.5.18-Python-2.7.11.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = 'rc3'
+
+cudaversion = '7.5.18'
+versionsuffix = '-CUDA-%s-Python-%%(pyver)s' % cudaversion
+
+homepage = 'https://github.com/BVLC/caffe'
+description = """
+Caffe is a deep learning framework made with expression, speed, 
+and modularity in mind. It is developed by the Berkeley Vision 
+and Learning Center (BVLC) and community contributors.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': False, 'pic': True}
```

Comment:
I think it slipped in while debugging the build process. I've removed it, and the software still builds fine.